src/main/views/cookies.njk

```diff
@@ -0,0 +1,74 @@
+{% extends "template.njk" %}
+{% from "./macros/common-components.njk" import goBack, submitButton, backToTopButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "moj/components/filter/macro.njk" import mojFilter %}
+
+{% block pageTitle %}
+    {{ title }}
+{% endblock %}
+
+{% block beforeContent %}
+    {{ super() }}
+{% endblock %}
+
+{% block content %}
+<h1 class="govuk-heading-l">{{ title }}</h1>
+<div class = "govspeak">
+<p>{{ cookiesExplanation1 }}</p>
+<p>{{ cookiesExplanation2 }}</p>
+<p>{{ cookiesExplanation3 }}</p>
+</div>
+
+{{ govukTable({
+  caption: "Essential Cookies",
```

Comment:
Believe this should be translated as well

Reply:
Good catch